lib/edge-runtime/missingParamCorrection.ts

```diff
@@ -16,6 +16,11 @@ export async function getMissingArgCorrections(
   corrections: { [param: string]: "ask user" | any };
   newSystemMessages: ChatGPTMessage[] | null;
 }> {
+  // Strip out unnecessary text from the system message
+  previousConversation[0].content = previousConversation[0].content
+    .split("You MUST exclusively use the functions listed below")[0]
+    .trim();
+
```

Comment:
Does this change the history for the chatbot too?

I.e. previousConversation[0].content is changed outside this function too?

Reply:
Minor side point: doing this by splitting makes this harder to maintain (i.e. can imagine the prompt changing which would break this, but this dependency not being easily spotted)